recipe/meta.yaml

```diff
@@ -18,18 +18,24 @@ build:
   script: CMAKE_GENERATOR="Ninja" {{ PYTHON }} -m pip install . -vv  # [not win]
   # Skip macOS build until https://github.com/Qiskit/qiskit-aer/issues/1689 is addressed
   skip: true  # [osx]
-  number: 1
+  number: 2
   script_env:
     - DISABLE_DEPENDENCY_INSTALL=ON
     - DISABLE_CONAN=ON
     - CMAKE_GENERATOR=Ninja
     - CMAKE_PREFIX_PATH=%PREFIX%  # [win]
+    - CUQUANTUM_ROOT=$PREFIX  # [linux64 and cuda_compiler_version != "None"]
+    - CUTENSOR_ROOT=$PREFIX  # [linux64 and cuda_compiler_version != "None"]
+    - AER_THRUST_BACKEND=CUDA  # [linux64 and cuda_compiler_version != "None"]
+    - AER_ENABLE_CUQUANTUM=ON  # [linux64 and cuda_compiler_version != "None"]
+    - CUQUANTUM_STATIC=ON  # [linux64 and cuda_compiler_version != "None"]
```

Comment:
This should not be set, because for most of conda-forge packages we want to link dynamically.
```suggestion
```